content/docs/faq-ajax.md

```diff
@@ -1,31 +1,31 @@
 ---
 id: faq-ajax
-title: AJAX and APIs
+title: AJAX dan API
 permalink: docs/faq-ajax.html
 layout: docs
 category: FAQ
 ---
 
-### How can I make an AJAX call? {#how-can-i-make-an-ajax-call}
+### Bagaimana cara membuat panggilan AJAX? {#how-can-i-make-an-ajax-call}
 
-You can use any AJAX library you like with React. Some popular ones are [Axios](https://github.com/axios/axios), [jQuery AJAX](https://api.jquery.com/jQuery.ajax/), and the browser built-in [window.fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).
+Anda bisa menggunakan semua AJAX *library* yang anda inginkan seperti halnya dengan React. Salah satu diantaranya yang populer yaitu [Axios](https://github.com/axios/axios), [jQuery AJAX](https://api.jquery.com/jQuery.ajax/), dan *browser* yang ter-integrasi [window.fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).
```

Comment:
> Anda dapat menggunakan semua _library_ AJAX yang diinginkan pada React. Beberapa yang populer yaitu Axios, jQuery AJAX, dan `window.fetch` bawaan browser.

Reply:
> Anda dapat menggunakan *library* AJAX apa saja untuk React.

Just a minor thing, *apa saja* for *any* instead of *semua*.

Reply:
```suggestion
Anda bisa menggunakan *library* AJAX apa saja yang anda inginkan dengan React. Beberapa yang populer yaitu [Axios](https://github.com/axios/axios), [jQuery AJAX](https://api.jquery.com/jQuery.ajax/), dan [window.fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) bawaan *browser*.
```